Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -357,6 +357,9 @@ YUI.add('ez-binarybase-editview', function (Y) {
          */
         _prepareDrop: function (e) {
             e.preventDefault();
+            if (this.get('isNotTranslatable')) {
```

Comment:
If I do that, the browsers reads the image as a file (instead of the page) `file://myfile.png`